pulp_2to3_migration/app/plugin/docker/migrator.py

```diff
@@ -267,3 +267,16 @@ async def _pre_save(self, batch):
                 # Manifests should have passed through ContentSaver stage already
                 man = Manifest.objects.filter(digest=related_man_id).first()
                 dc.content.tagged_manifest = man
+
+    async def _post_save(self, batch):
+        """
+        Remove tag if it  points to a tagged_manifest=null
+
+        Args:
+            batch (list of :class:`~pulpcore.plugin.stages.DeclarativeContent`): The batch of
+                :class:`~pulpcore.plugin.stages.DeclarativeContent` objects to be saved.
+
+        """
+        for dc in batch:
+            if type(dc.content) == Tag and not dc.content.tagged_manifest:
```

Comment:
Not critical here so it;s up to you, but I'd vote for the `isinstance` instead of `type`. Taught myself not to use `type` in Python at all. Just a as a good habit, so it doesn't bite in certain situations.